defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+apt_sources_ubuntu: {}
+apt_sources_ubuntu_mirror: http://de.archive.ubuntu.com/
+apt_sources_ubuntu_enable_sources: False
+apt_sources_ubuntu_components:
+  - main
+  - universe
+apt_sources_ubuntu_security_repo: True
```

Comment:
Please mention in the readme that the default values can be adapted using these variables.
E.g.
```
All defaults can be adapted by setting the respective variable, prepended by `apt_sources_ubuntu_`, or `apt_sources_debian_`.

Reply:
You can simple use `apt_sources_ubuntu` or `apt_sources_debian` dict. There is no need to set a `apt_sources_*_*` default value manuell.

Reply:
I'm sure there is a use case for it, maybe someone wants a different default value. Also, not all Ansible users have an inventory script that merges dicts in a smart way.
I don't mean that you describe every variable, just the hint that the defaults can also be configured, like I wrote it above, or you could link to the defaults file instead.